liquibase-extension-testing/src/main/java/liquibase/extension/testing/testsystem/core/MSSQLTestSystem.java

```diff
@@ -35,7 +37,7 @@ protected Runnable requireLicense() {
     public String getConnectionUrl() {
         final JdbcDatabaseContainer container = ((DockerDatabaseWrapper) wrapper).getContainer();
 
-        return "jdbc:sqlserver://" + container.getHost() + ":" + container.getMappedPort(MSSQLServerContainer.MS_SQL_SERVER_PORT)+";databaseName="+getCatalog()+";trustServerCertificate=true;encrypt=false";
+        return "jdbc:sqlserver://" + container.getHost() + ":" + container.getMappedPort(MSSQLServerContainer.MS_SQL_SERVER_PORT)+";databaseName="+getCatalog()+";encrypt=false";
```

Comment:
The encrypt=false is needed here because this is a URL we are building up ourselves vs. relying on what the testcontainers system is going to give us.

It doesn't need the trustServerCertificate=true setting that was removed, but the encrypt=false is needed.

Reply:
Makes sense, @nvoxland. Thanks!